src/main/java/com/codejoust/main/dto/problem/CreateProblemTagRequest.java

```diff
@@ -0,0 +1,10 @@
+package com.codejoust.main.dto.problem;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class CreateProblemTagRequest {
+    private String name;
```

Comment:
Yeah I agree - not sure if that would entail creating a public vs. private setting for problems, but regardless it would help the teacher organize their own problems.